### PR TITLE
fixes chewable trash items not spawning

### LINKED ISSUE #9404: Non-Smokables Used Up
Uxsing up non smokeables, such as gum or suckers or chewing tobacco or the such doesn't create anything when it's used up, just vanishes, chewed up wads of gum or pospicle sticks or whatever you get from used up chewing tobacco.

Expected to make gross garbage to ignore or stick to people.

Nothing at all was made instead.

### PR DESCRIPTION
**Pull request by alsoandanswer**
title

Fixes #9404

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
!bugfix

File: code/game/objects/items/weapons/chewables.dm [hunk 1]
```diff
@@ -84,28 +84,28 @@ obj/item/clothing/mask/chewable/Destroy()
 	desc = "A disgusting spitwad."
 	icon = 'icons/obj/clothing/masks.dmi'
 	icon_state = "spit-chew"
+	slot_flags = SLOT_EARS | SLOT_MASK
 
-/obj/item/clothing/mask/chewable/proc/spitout(var/mob/user, var/no_message)
+/obj/item/clothing/mask/chewable/proc/spitout(var/transfer_color = 1, var/no_message = 0, var/mob/living/M)
```
Comment (mikomyazaki) [inline code/game/objects/items/weapons/chewables.dm:89]: M doesn't need to be an argument, you can just define it on line 99 as it was previously.

Comment (alsoandanswer) [reply code/game/objects/items/weapons/chewables.dm:89]: i like this better as it's in line with cigarette code

Comment (skull132) [reply code/game/objects/items/weapons/chewables.dm:89]: No but that's bad programming.

`M` should not be a parameter if you don't care about it and just write over it.

**Comment by mikomyazaki** [comment]
They should both be like in https://github.com/Aurorastation/Aurora.3/pull/9418/commits/eab00f3a55923d8b7c61e1835ef8f080ba8c3b97

M does not need to be defined until after the ismob() checks in both cases.

**Pull request merged**